addon/templates/components/uni-datepicker.hbs

```diff
@@ -1,7 +1,7 @@
 {{#power-calendar
   center=center
   selected=selected
-  onCenter=(action onCenter value="moment")
+  onCenterChange=(action onCenter value="moment")
```

Comment:
Did you make sure our current `ember-power-calendar` version supports this API? Because this seems to me a breaking change and it might not be the only one

Reply:
I did, that's the reason this change was required. I also tested the patch in a separate project.

This might have been incorrect from the beginning, but this patch adds an assert to validate it:

`v0.7.3` (our current, outdated, version):
https://github.com/cibernox/ember-power-calendar/commit/290461e795db970735bd212e4f1f5855abce4934

Also from the changelog:
```
## 0.7.3
- [ENHANCEMENT] Add helpful assertions when a calendar receives an invalid date as center and when the user tries to change the center of a calendar without an `onCenterChange` action.
```

There's usage of `onCenterChange` all the way back to `0.1.x`.